L/Libxc/Libxc_GPU/build_tarballs.jl

```diff
@@ -56,10 +78,68 @@ dependencies = [
+                                "0b85f7eee17788abbd170b0b493c74ce2e9fd5a9604461b99c2c378165e1083b"),
+                 )
+        elseif cuda_ver == "12.2"
+            # See https://developer.download.nvidia.com/compute/cuda/redist/redistrib_12.2.1.json
+            push!(platform_sources,
+                  ArchiveSource("https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/linux-x86_64/cuda_nvcc-linux-x86_64-12.2.128-archive.tar.xz", 
+                                "018086c6bce5868451b0d30c74fd78826e15a2af0e9d891c1843bc2c3884bdec"),
+                 )
+        elseif cuda_ver == "12.3"
+            # See https://developer.download.nvidia.com/compute/cuda/redist/redistrib_12.3.1.json
+            push!(platform_sources,
+                  ArchiveSource("https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/linux-x86_64/cuda_nvcc-linux-x86_64-12.3.103-archive.tar.xz",
+                                "ae86efce6a69e99c55def1203157aee4bf71d6e5f8423c2a8d69a0e97036e9db"),
+                 )
+        elseif cuda_ver == "12.4"
+            # See https://developer.download.nvidia.com/compute/cuda/redist/redistrib_12.4.1.json
+            push!(platform_sources,
+                  ArchiveSource("https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/linux-x86_64/cuda_nvcc-linux-x86_64-12.4.131-archive.tar.xz",
+                                "7ffba1ada0e4b8c17e451ac7a60d386aa2642ecd08d71202a0b100c98bd74681"),
+                 )
+        elseif cuda_ver == "12.5"
+            # See https://developer.download.nvidia.com/compute/cuda/redist/redistrib_12.5.1.json
+            push!(platform_sources,
+                  ArchiveSource("https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/linux-x86_64/cuda_nvcc-linux-x86_64-12.5.82-archive.tar.xz",
+                                "ded05fe3c8d075c6c1bf892005d3c50bde3eceaa049b879fcdff6158e068e3be"),
+                 )
+        elseif cuda_ver == "12.6"
+            # See https://developer.download.nvidia.com/compute/cuda/redist/redistrib_12.6.1.json
+            push!(platform_sources,
+                  ArchiveSource("https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/linux-x86_64/cuda_nvcc-linux-x86_64-12.6.68-archive.tar.xz",
+                                "b672d0c36a27ea4577536725713064c9daa5d7378ac85877bc847ca9a46b2645"),
+                 )
+        elseif cuda_ver == "12.8"
+            # See https://developer.download.nvidia.com/compute/cuda/redist/redistrib_12.8.1.json
+            push!(platform_sources,
+                  ArchiveSource("https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/linux-x86_64/cuda_nvcc-linux-x86_64-12.8.93-archive.tar.xz",
+                                "9961b3484b6b71314063709a4f9529654f96782ad39e72bf1e00f070db8210d3"),
+                 )
+        end
```

Comment:
Can you please move this to `platforms/cuda.jl` as a function called, say, `cuda_nvcc_redist_source(cuda_ver, arch)` which gives you the source entry for the given cuda version, so that here we can just do
```julia
    if arch(platform) == "aarch64"
        push!(platform_sources, cuda_nvcc_redist_source(cuda_ver, "x86_64"))
    end
```
?  In this way we wouldn't have to repeat the same everywhere.

Reply:
This is a nice suggestion, and I implemented it. As of now, I only added the `x86_64` redist. I am happy to extend this to `aarch64`, but I am a little confused. Both `linux-sbsa` and `linux-aarch64` redists are available for download. In `Reactant`, I see that you use `linux-sbsa`  in conjunction with `aarch64`. Any idea on how I can generalize this?

Reply:
I think we need not worry about this right now, at the moment we need this only to get the x86_64 binaries for cross-compilation, so in the short term this is useful only with x86_64 architecture anyway.  The extra argument `arch` is for allowing future extension, but not critical right now.